library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -140,4 +141,57 @@ class FormatUtilsTest {
             """.trimIndent()
         assertThat(FormatUtils.formatXml(xml)).isEqualTo(expected)
     }
+
+    @Test
+    fun testBasicCurlCommand() {
+        getRequestMethods().forEach { method ->
+            val transcation = TestTransactionFactory.createTransaction(method)
+            val curlcommand = FormatUtils.getShareCurlCommand(transcation)
+            val expectedCurlCommand = "curl -X $method http://localhost/getUsers"
+            assertThat(curlcommand).isEqualTo(expectedCurlCommand)
+        }
+    }
+
+    @Test
+    fun testCurlCommandWithHeaders() {
+        /**
+         * populate few dummy headers
+         */
+        val httpHeaders = ArrayList<HttpHeader>()
+        for (i in 0 until 5) {
+            httpHeaders.add(HttpHeader("name$i", "value$i"))
+        }
+        val dummyHeaders = JsonConverter.instance.toJson(httpHeaders)
+
+        getRequestMethods().forEach { method ->
+            val transcation = TestTransactionFactory.createTransaction(method)
+            transcation.requestHeaders = dummyHeaders
+            val curlcommand = FormatUtils.getShareCurlCommand(transcation)
+            var expectedCurlCommand = "curl -X $method"
+            httpHeaders.forEach { header ->
+                expectedCurlCommand += " -H \"${header.name}: ${header.value}\""
+            }
+            expectedCurlCommand += " http://localhost/getUsers"
+            assertThat(curlcommand).isEqualTo(expectedCurlCommand)
+        }
+    }
+
+    @Test
+    fun testCurlCommandWithRequestBody() {
+        /**
+         * Test POST and PUT with request body
```

Comment:
You should change test name to something more meaningful instead of adding such comment

Reply:
agreed changed the name to `testCurlPostAndPutCommandWithRequestBody`